### ISSUE TITLE
[Feature Request] Permissions by DNN roles and with more options

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#1288]
Hey there.

I'm having a hard time trying to figure out how permissions work.
Giving users security roles in DNN like edit module/page just to be able to edit app data doesn't work for me.

I need to give users strict permissions to add or edit any data on the view/content in that app/module, but they cannot have edit permissions to the page or the module itself (only the data in it, like FnL allows).

Basically, the content tool bars or list tool bars should be visible to some select DNN users with no security access (more like a social group).

Is this possible?

Best regards,
João Gomes

**Comment by iJungleboy** [comment]
I think this is what you're looking for :)
https://github.com/2sic/2sxc/wiki/Concept-Hide-Advanced-Features

**Comment by eXistenZe** [comment]
2sxc Designers group is not possible. Way too many groups with different accesses to different apps.

Custom toolbars do not seem to render...
I used 
`<ul class="sc-menu" toolbar='{"contentType":"farm_ad","action":"new"}'></ul>`
and I can see the code in the page source, but the button itself is not rendered for non admins.

I tried giving full permissions to the content and view to everyone (security level: view permissions and even anonymous), but the buttons do not show...

I'm trying this code:
```
@Dnn.User.IsInRole("Editores de fármacos")<br>
@if (@Dnn.User.IsInRole("Editores de fármacos") == true)
    {
    <div class="sc-element">
    <h1>Fármacos adulto</h1>
    @Edit.Toolbar(ListContent)
    </div>
    <br>
    <ul class="sc-menu" toolbar='{"contentType":"farm_ad","action":"new"}'></ul>
    <br>
    }
```

And I can see the if clause returns true (the h1 is rendered), but toolbar and buttons simply do not show...
